core/xim/xim.lua

```diff
@@ -42,8 +43,18 @@ end
 --      xmake l xim.lua vscode -i
 --      xmake l xim.lua -r vscode -y
```

Comment:
Not recommend a random order in params input, sometimes in nested structure, it has semantic vagueness.